code/modules/spells/spell_types/construct_spells.dm

```diff
@@ -277,4 +310,28 @@
 /obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/golem
 	charge_max = 800
 	jaunt_in_type = /obj/effect/temp_visual/dir_setting/cult/phase
-	jaunt_out_type = /obj/effect/temp_visual/dir_setting/cult/phase/out
\ No newline at end of file
+	jaunt_out_type = /obj/effect/temp_visual/dir_setting/cult/phase/out
+
+
+/obj/effect/proc_holder/spell/dumbfire/juggernaut
+	name = "Gauntlet Echo"
+	desc = "Channels energy into your gauntlet - firing its essence forward in a slow-moving but devastating blow."
+	proj_icon_state = "cursehand0"
```

Comment:
personally I'd use a lower-alpha version of the juggernaut's own sprite, with appropriate lore tweaks

Reply:
I've tried several different icons, including a homebrew. The shadowhand looks pretty good (unless you fire it west). I could try your approach but im overall pretty happy with the hand's appearance. I was considering making the projectile dense to act as a mobile shield though... that would fit your appearance better.

Reply:
I'd personally think a half-alpha, fully red juggernaut sprite would look cool.